lib/godmin/generators/named_base.rb

```diff
@@ -7,6 +7,14 @@ class NamedBase < Base
 
       private
 
+      def full_class_name
```

Comment:
Are you sure this is desirable?

Unless I'm mistaken, that code would get "expanded" into something similar to this

```ruby
namespace, name = Admin, 'AdminUser'
(namespace.name.split('::').map(&:underscore) + name.classify.deconstantize.split("::").map(&:undescore)).join('/').classify
```

That's quite a lot of transformation being chained here.